client/post-editor/editor-sharing/publicize-connection.jsx

```diff
@@ -163,6 +167,35 @@ export class EditorSharingPublicizeConnection extends React.Component {
 		);
 	};
 
+	/**
+	 * If a connection needs reauthentication, display a warning linked to Sharing settings page so users can disconnect and reconnect.
+	 *
+	 * @returns {object} Warning about connection.
+	 */
+	renderMustReauthConnection = () => {
+		const { connection, siteSlug } = this.props;
+		if ( ! connection || connection.status !== 'must_reauth' ) {
```

Comment:
There are so many places where we check whether a connection needs reauth. I wonder if we can consolidate these using a common helper.

Reply:
Overkill for only two uses in this file and this is the only negative one